New Bank/NewBank/newbank/server/NewBankClientHandler.java

```diff
@@ -69,4 +67,21 @@ public void run() {
 		}
 	}
 
+	public boolean loggedIn() {
+		try {
+			String responce = "FAIL"; // initialise responce string
+			while (responce!="SUCCESS"){ // while loop until success is achieved, will not bring up login entry until then
+				// ask if login or signup
+				out.println("LOGIN or SIGNUP");
+				String request = in.readLine();
+				responce = bank.processRequest(request);
+				out.println(responce);
+			}
+			return true;
+		} catch (IOException | NumberFormatException e) {
+			e.printStackTrace();
+		}
+		return false;
+	}
+
```

Comment:
loggedIn() method should be the same as the original LOGIN try/catch but with return true/false added.